plugins/apache_http.yaml

```diff
@@ -36,14 +36,31 @@ parameters:
      - beginning
      - end
     default: end
+  - name: log_format
+    label: Log Format
+    description: |2
+      Default is unmodifed log_format settings for Apache HTTPD access.log and error.log
 
+      Observiq Log Format expects the following formats for the access and error logs. This format will generate a json log entry. This log_format can be modified to meet your requirements as long as key names do not change and it is valid json.
+      To setup edit apache2.conf. Add both log formats below the default log formats. Edit sites-available configurations: modify access log to use observiq format instead of combined.
```

Comment:
I think this should be re-worded to be more generic. Our instructions are for Ubuntu, the process is different on Centos.

```
To setup, edit Apache's configuration. Add both log formats to the logging section. On Debian, update all `sites-enabled` configurations to include `CustomLog ${APACHE_LOG_DIR}/access.log observiq`. On CentOS, update Apache's configuration to include `CustomLog "logs/access_log" observiq`.
```